snippets/facets.liquid

```diff
@@ -111,7 +112,18 @@
             <script src="{{ 'show-more.js' | asset_url }}" defer="defer"></script>
             {% comment %} Filters for both horizontal and vertical filter {% endcomment %}
             {%- for filter in results.filters -%}
-              {%- assign total_active_values = total_active_values | plus: filter.active_values.size -%}
+              {% liquid
+                assign total_active_values = total_active_values | plus: filter.active_values.size
+                case filter.display_style
+                  when 'swatch', 'thumbnail'
+                    assign has_visual_representation = true
+                    assign visual_layout_class = 'facets-layout-grid facets-layout-grid--' | append: filter.display_style
+                  else
+                    assign has_visual_representation = false
+                    assign visual_layout_class = 'facets-layout-list'
+                endcase
+              %}
+
```

Comment:
(I think the right place to put this comment is around line 145 but I can't reach ✋)
It could be just something wrong with my store, but when 🎩 ing, the filter `selected` count doesn't seem to work?
![image](https://github.com/Shopify/dawn/assets/107708248/eb56f187-9718-4b77-9d59-3fb8b223701c)
It was working on Dawn-11.0.0

Reply:
Yeah, blame @patrickracicot :D https://github.com/Shopify/dawn/pull/3048 - it's fixed in main, I'll rebase and squash in a bit.

Reply:
Ohh, missed that whole situation, I'm glad it wasn't Just Me 😆